### DISCUSSION TITLE
Unable to log in to Google or nextCloud

### DISCUSSION THREAD

**Discussion by Topbanaana** [SpiralLinux/SpiralLinux-project#107] [Q&A]
In my Budgie Desktop Settings/Online subscriptions I add my credentials as usual but nothing happens - I am unable to connect to my Google or nextCloud box. Anybody with same issue here?

**Comment by geckolinux** [comment]
Hi there, sorry for the slow reply.
I admit I haven't tested that functionality. It might just be a matter of a missing package. Could you please try installing `gnome-online-accounts` ?

**Reply by Topbanaana** [reply]
Looks like that did the trick. Thanks.

**Reply by geckolinux** [reply]
Great, I'll include it in the next release.

**Comment by geckolinux** [comment]
Implemented in SpiralLinux 11.220925 release.
